extensions/amp-analytics/0.1/vendors.js

```diff
@@ -50,6 +50,7 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'externalReferrer': 'EXTERNAL_REFERRER',
       'firstContentfulPaint': 'FIRST_CONTENTFUL_PAINT',
       'firstViewportReady': 'FIRST_VIEWPORT_READY',
+      'htmlAttr': 'HTML_ATTR',
```

Comment:
let's not have this alias, given this is a very specific feature, most publishers/vendors won't care.

Reply:
Done